### PR TITLE
Add appsidebar close button unique class

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: src/mixins/actionGlobal.js [hunk 1]
```diff
@@ -23,6 +23,8 @@
 import Vue from 'vue'
 
 export default {
+	inheritAttrs: false,
```
Comment (danxuliu) [inline src/mixins/actionGlobal.js]: Why is this needed to add the unique class? (It is probably pretty obvious, but remember that I am new to Vue :-P )

Comment (skjnldsv) [reply src/mixins/actionGlobal.js]: Argh, nice catch!
This is a wrong part of this pr :see_no_evil:

Comment (danxuliu) [reply src/mixins/actionGlobal.js]: Now I feel like a pro B-) :-P
